Java/import-api-sample/src/main/java/com/perfecto/reporting/sample/api/ImportTestMobile.java

```diff
@@ -11,16 +11,16 @@
 import com.perfecto.reportium.model.Project;
 import com.perfecto.reportium.test.TestContext;
 import com.perfecto.reportium.test.result.TestResultFactory;
-
 import java.net.URI;
 
 public class ImportTestMobile {
-
-    private static final String REPORTIUM_URL = "https://[MY_COMPANY_ID].reporting.perfectomobile.com";
-    private static final String SECURITY_TOKEN = "[MY_SECURITY_TOKEN]";
+    private static final String MY_COMPANY_ID = "company-id";
+    private static final String PERFECTO_TOKEN_KEY = "security-token";
```

Comment:
is this the header name?
call it SECURITY_TOKEN_HEADER_NAME ?

Reply:
PERFECTO_TOKEN_KEY is the name of the env var the suer needs to give to provide the key from cli.

Reply:
This is not the term used in documentation